### PR TITLE
[PhpUnitBridge] fix disabling DeprecationErrorHandler using phpunit.xml file

### PR DESCRIPTION
**Pull request by soerenbernstein**
Fixing #28519

| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | don't know
| Fixed tickets | #28519
| License       | MIT

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php [hunk 1]
```diff
@@ -54,7 +54,11 @@ public static function register($mode = 0)
             if (false === $mode) {
                 $mode = getenv('SYMFONY_DEPRECATIONS_HELPER');
             }
-            if (DeprecationErrorHandler::MODE_WEAK !== $mode && DeprecationErrorHandler::MODE_WEAK_VENDORS !== $mode && (!isset($mode[0]) || '/' !== $mode[0])) {
+            if (DeprecationErrorHandler::MODE_DISABLED !== $mode 
+                && DeprecationErrorHandler::MODE_WEAK !== $mode 
+                && DeprecationErrorHandler::MODE_WEAK_VENDORS !== $mode 
+                && (!isset($mode[0]) || '/' !== $mode[0])
+               ) {
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php]: indentation should be one level down here

File: src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php [hunk 2]
```diff
@@ -54,7 +54,11 @@ public static function register($mode = 0)
             if (false === $mode) {
                 $mode = getenv('SYMFONY_DEPRECATIONS_HELPER');
             }
-            if (DeprecationErrorHandler::MODE_WEAK !== $mode && DeprecationErrorHandler::MODE_WEAK_VENDORS !== $mode && (!isset($mode[0]) || '/' !== $mode[0])) {
+            if (DeprecationErrorHandler::MODE_DISABLED !== $mode
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php]: please remove trailing spaces  (see fabbot failure)

**Comment by nicolas-grekas** [comment]
@soerenbernstein I suppose you confirm this fixes the issue and allows disabling the bridge using the phpunit.xml file?

**Comment by soerenbernstein** [comment]
Yes. And it will actually set the mode to "disabled" for *future* use.

**Comment by nicolas-grekas** [comment]
Thank you @soerenbernstein.

**Pull request merged**